proposals/78:upgrade-17/synthetic-chain-excerpt.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * @file work-around: importing @agoric/synthetic-chain hangs XXX
```

Comment:
??? I wonder what that's about.

But I guess it passed review in agoric-sdk, so... oh well.